rest-api/dao/calendar_dao.py

```diff
@@ -0,0 +1,10 @@
+from dao.base_dao import BaseDao
+from model.calendar import Calendar
+
+class CalendarDao(BaseDao):
```

Comment:
Please add a simple insert here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/test/unit_test/database_test.py#L22

This just verifies that our model classes generally work.

Reply:
Done.